Gemfile

```diff
@@ -38,6 +38,8 @@ gem 'gpx'
 gem "activeadmin", git: "https://github.com/activeadmin/activeadmin"
 gem "inherited_resources", git: "https://github.com/activeadmin/inherited_resources"
 gem "activerecord-import"
+gem "delayed_job_active_record"
```

Comment:
Is there special purpose of choosing delayed_job and not [sidekiq](https://github.com/mperham/sidekiq) for example?